### PR TITLE
Project Readme Updates

### PR DESCRIPTION
**Pull request by kiritigowda**
All Read updates and bug fixes.

### CODE REVIEW

**Comment by kiritigowda** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 24
- Added 6
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6056700&bid=19840675)

File: apps/mivisionx_openvx_classifier/README.md [hunk 1]
```diff
@@ -55,35 +58,39 @@ Usage: ./classifier
...
 **Note:** MIVisionX installs all the model compiler scripts in `/opt/rocm/mivisionx/model_compiler/python/` folder
 
+ + Convert the pre-trained caffemodel into AMD NNIR model:
+
+  ``` 
+  % python /opt/rocm/mivisionx/model_compiler/python/caffe_to_nnir.py <net.caffeModel> <nnirOutputFolder> --input-dims <n,c,h,w> [--verbose <0|1>]
+  ```
 
-	* Convert the pre-trained caffemodel into AMD NNIR model:
+  Sample:
```
Comment (kiritigowda) [inline apps/mivisionx_openvx_classifier/README.md:75]: Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, add 2 spaces](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6056700)

File: model_compiler/README.md [hunk 2]
```diff
@@ -247,50 +271,52 @@ Usage: anntest <weights.bin> [<input-data-file(s)> [<output-data-file(s)>]]]
...
 
 |Networks|Caffe|ONNX|NNEF|
 |--------|-----|----|----|
-|AlexNet||&#9745;|&#9745;|
-|Caffenet||&#9745;||
-|DenseNet||&#9745;||						
-|Googlenet|&#9745;|&#9745;|&#9745;|		
-|Inception-V1||&#9745;|&#9745;|			
-|Inception-V2||&#9745;|&#9745;|			
+|AlexNet||&#9745; |&#9745; |
```
Comment (kiritigowda) [inline model_compiler/README.md:291]: Codacy found an issue: [[table-cell-padding] Cell should be compact](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6056700)

File: apps/mivisionx_openvx_classifier/README.md [hunk 3]
```diff
@@ -55,35 +58,39 @@ Usage: ./classifier
...
+  ```
 
-	**Note:** The weights.bin file will be generated inside the OpenVX folder and you can use that as an input for this project.
+  Sample:
+
+    ``` 
+    % python /opt/rocm/mivisionx/model_compiler/python/nnir_to_openvx.py VGG16_NNIR VGG16_OpenVX
+    ```
+
+ **Note:** The weights.bin file will be generated inside the OpenVX folder and you can use that as an input for this project.
```
Comment (kiritigowda) [inline apps/mivisionx_openvx_classifier/README.md:93]: Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, add 2 spaces](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6056700)

File: README.md [hunk 4]
```diff
@@ -355,38 +350,35 @@ sudo docker pull mivisionx/ubuntu-16.04
...
+  ##### 1. Map host directory on the docker image
+  
+  * map the localhost directory to be accessed on the docker image.
+  * use `-v` option with docker run command: `-v {LOCAL_HOST_DIRECTORY_PATH}:{DOCKER_DIRECTORY_PATH}`
+  * usage:
+    ``` 
+    sudo docker run -it -v /home/:/root/hostDrive/ --device=/dev/kfd --device=/dev/dri --cap-add=SYS_RAWIO --device=/dev/mem --group-add video --network host mivisionx/ubuntu-16.04
+    ```
+  
+  ##### 2. Display option with docker
```
Comment (kiritigowda) [inline README.md:363]: Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, add 2 spaces](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6056700)

File: README.md [hunk 5]
```diff
@@ -355,38 +350,35 @@ sudo docker pull mivisionx/ubuntu-16.04
...
-export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/opt/rocm/mivisionx/lib
-runvx /opt/rocm/mivisionx/samples/gdf/canny.gdf 
-```
+  **Note:** 
+  ##### 1. Map host directory on the docker image
+  
+  * map the localhost directory to be accessed on the docker image.
+  * use `-v` option with docker run command: `-v {LOCAL_HOST_DIRECTORY_PATH}:{DOCKER_DIRECTORY_PATH}`
+  * usage:
+    ```
```
Comment (kiritigowda) [inline README.md:359]: Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, add 2 spaces](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6056700)

File: apps/mivisionx_openvx_classifier/README.md [hunk 6]
```diff
@@ -55,35 +58,39 @@ Usage: ./classifier
...
 
 Here is the sample download [link](https://github.com/SnailTyan/caffe-model-zoo) that contains all the prototxt: 
 
 2. Use [MIVisionX Model Compiler](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/tree/master/model_compiler#neural-net-model-compiler--optimizer) to extract weights.bin from the pre-trained caffe models
+
 **Note:** MIVisionX installs all the model compiler scripts in `/opt/rocm/mivisionx/model_compiler/python/` folder
 
+ + Convert the pre-trained caffemodel into AMD NNIR model:
+
+  ```
```
Comment (kiritigowda) [inline apps/mivisionx_openvx_classifier/README.md:71]: Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, add 2 spaces](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6056700)

**Comment by kiritigowda** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 24
- Added 6
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6056700&bid=19840675)

File: apps/mivisionx_openvx_classifier/README.md [hunk 7]
```diff
@@ -55,35 +58,39 @@ Usage: ./classifier
...
 
-	**Note:** The weights.bin file will be generated inside the OpenVX folder and you can use that as an input for this project.
+  Sample:
+
+    ``` 
+    % python /opt/rocm/mivisionx/model_compiler/python/nnir_to_openvx.py VGG16_NNIR VGG16_OpenVX
+    ```
+
+  **Note:** The weights.bin file will be generated inside the OpenVX folder and you can use that as an input for this project.
```
Comment (kiritigowda) [inline apps/mivisionx_openvx_classifier/README.md:93]: Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, add 2 spaces](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6056700)

File: apps/mivisionx_openvx_classifier/README.md [hunk 8]
```diff
@@ -55,35 +58,39 @@ Usage: ./classifier
...
 
 Here is the sample download [link](https://github.com/SnailTyan/caffe-model-zoo) that contains all the prototxt: 
 
 2. Use [MIVisionX Model Compiler](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/tree/master/model_compiler#neural-net-model-compiler--optimizer) to extract weights.bin from the pre-trained caffe models
+
 **Note:** MIVisionX installs all the model compiler scripts in `/opt/rocm/mivisionx/model_compiler/python/` folder
 
+  + Convert the pre-trained caffemodel into AMD NNIR model:
+
+  ```
```
Comment (kiritigowda) [inline apps/mivisionx_openvx_classifier/README.md:71]: Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, add 2 spaces](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6056700)

File: README.md [hunk 9]
```diff
@@ -355,38 +350,36 @@ sudo docker pull mivisionx/ubuntu-16.04
...
+  ##### 1. Map host directory on the docker image
+  
+  * map the localhost directory to be accessed on the docker image.
+  * use `-v` option with docker run command: `-v {LOCAL_HOST_DIRECTORY_PATH}:{DOCKER_DIRECTORY_PATH}`
+  * usage:
+  ``` 
+  sudo docker run -it -v /home/:/root/hostDrive/ --device=/dev/kfd --device=/dev/dri --cap-add=SYS_RAWIO --device=/dev/mem --group-add video --network host mivisionx/ubuntu-16.04
+  ```
+  
+  ##### 2. Display option with docker
```
Comment (kiritigowda) [inline README.md:363]: Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, add 2 spaces](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6056700)

File: model_compiler/README.md (hunk 2 above)
Comment (kiritigowda) [inline model_compiler/README.md:291]: Codacy found an issue: [[table-cell-padding] Cell should be compact](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6056700)

File: apps/mivisionx_openvx_classifier/README.md [hunk 10]
```diff
@@ -55,35 +58,39 @@ Usage: ./classifier
...
 **Note:** MIVisionX installs all the model compiler scripts in `/opt/rocm/mivisionx/model_compiler/python/` folder
 
+  + Convert the pre-trained caffemodel into AMD NNIR model:
+
+  ``` 
+  % python /opt/rocm/mivisionx/model_compiler/python/caffe_to_nnir.py <net.caffeModel> <nnirOutputFolder> --input-dims <n,c,h,w> [--verbose <0|1>]
+  ```
 
-	* Convert the pre-trained caffemodel into AMD NNIR model:
+  Sample:
```
Comment (kiritigowda) [inline apps/mivisionx_openvx_classifier/README.md:75]: Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, add 2 spaces](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6056700)

File: README.md [hunk 11]
```diff
@@ -355,38 +350,36 @@ sudo docker pull mivisionx/ubuntu-16.04
...
-export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/opt/rocm/mivisionx/lib
-runvx /opt/rocm/mivisionx/samples/gdf/canny.gdf 
-```
+  **Note:** 
+  ##### 1. Map host directory on the docker image
+  
+  * map the localhost directory to be accessed on the docker image.
+  * use `-v` option with docker run command: `-v {LOCAL_HOST_DIRECTORY_PATH}:{DOCKER_DIRECTORY_PATH}`
+  * usage:
+  ```
```
Comment (kiritigowda) [inline README.md:359]: Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, add 2 spaces](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6056700)
